static/src/js/middleware/metrics/events.js

```diff
@@ -11,6 +11,7 @@ const { dataLayer = [] } = window
 
 /**
 * Pushes a virtualPageView event on the dataLayer. Only fires on PUSH events.
+$ These `PUSH` events only fire during page transitions
```

Comment:
Is these supposed to be a `$` instead of `*` in the comment?

Reply:
No, good catch thanks